edge-functions/next-news/pages/_middleware.ts

```diff
@@ -0,0 +1,7 @@
+import { NextResponse as Response } from "next/server";
+
+export default function middleware(req) {
+  if (req.nextUrl.pathname === "/") {
+    return Response.rewrite("/news/1");
+  }
+}
```

Comment:
This part should be included on the readme, alongside a description of why are we using middleware in this example.